pkg/dockerfile/generator.go

```diff
@@ -42,6 +42,7 @@ coverage.xml
 .pytest_cache
 .hypothesis
 `
+const CythonCFlags = "CFLAGS=\"-O3 -march=native -ffast-math -funroll-loops -fno-strict-aliasing -flto -mtune=native -S\""
```

Comment:
Given this is an environment variable, would perhaps be easier/cleaner to set it with `ENV` and (if necessary) unset it at the end, rather than interpolating it in multiple places?

Reply:
I'll add this to the preamble `ENV` commands instead, this will ensure these settings are applied to any compilation process, I can't think of a reason why that wouldn't be ultimately desirable.

Reply:
I opt for wrapping pip installs in the set/unset of this ENV var to keep the blast area smaller